infra/embedding_step_functions/components/word_workflow.py

```
@@ -154,23 +167,29 @@ def _create_ingest_workflow(self) -> StateMachine:
-        arns[5] = embedding-mark-batches-complete
+        arns_and_bucket[0] = embedding-list-pending
+        arns_and_bucket[1] = embedding-word-poll
+        arns_and_bucket[2] = embedding-vector-compact
+        arns_and_bucket[3] = embedding-normalize-poll-batches
+        arns_and_bucket[4] = embedding-split-chunks
+        arns_and_bucket[5] = embedding-create-chunk-groups
+        arns_and_bucket[6] = embedding-mark-batches-complete
+        arns_and_bucket[7] = batch_bucket_name
         """
+        arns = arns_and_bucket[:-1]
+        batch_bucket_name = arns_and_bucket[-1]
```

Ruff (0.14.5)

Local variable `batch_bucket_name` is assigned to but never used

Remove assignment to unused variable `batch_bucket_name`

(F841)